.buildkite/bin/create_tag_and_update_branch.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+# $1 version
+# $2 Push to branch option (TRUE|TAG_ONLY)
+
+git config user.email "nativeappsdev@rokt.com"
```

Comment:
Can we move this to the secret manager?

Reply:
Don't think its a secret since it'll show up in the git log
![image](https://github.com/ROKT/rokt-sdk-ios/assets/122851605/91726d76-5326-4355-8df6-8edf7aa8b829)